### DISCUSSION TITLE
Detect click on overlapping highlights

### DISCUSSION THREAD

**Discussion by matteoappload** [readium/kotlin-toolkit#588] [Q&A]
Hi, i'd like to detect when a user clicks on an highlight, but i want to show a list of highlights if there are many of them overlapping (even if only partially). I can't figure out how to achieve this.

**Comment by rpiero99** [comment]
up, i've the same problem...

**Comment by mickael-menu** [comment]
This is not currently not supported and you would need to modify the toolkit for that.
